battle/boss.py

```diff
@@ -1476,48 +1479,48 @@ def seymour_flux():
                     else:
                         battle.main.attack("none")
                 elif screen.turn_tidus():
-                    print("Tidus' turn. Stage:", stage)
+                    logger.debug(f"Tidus' turn. Stage: {stage}")
                     if stage < 3:
                         battle.main.tidus_haste("down", character=1)
                     elif lastHP > 3500:
                         battle.main.attack("none")
                     else:
                         battle.main.defend()
                 elif screen.turn_auron():
-                    print("Auron's turn. Swap for Rikku and overdrive.")
+                    logger.debug("Auron's turn. Swap for Rikku and overdrive.")
                     battle.main.buddy_swap_rikku()
-                    print("Rikku overdrive")
+                    logger.debug("Rikku overdrive")
                     battle.main.rikku_full_od("Flux")
                 else:
-                    print("Non-critical turn. Defending.")
+                    logger.debug("Non-critical turn. Defending.")
                     battle.main.defend()
             elif memory.main.diag_skip_possible():
                 xbox.tap_b()
     memory.main.click_to_control()
     if memory.main.get_slvl_yuna() - yunaXP == 15000:
         game_vars.flux_overkill_success()
-    print("-----------------------------")
-    print("Flux Overkill:", game_vars.flux_overkill())
-    print("Seymour Flux battle complete.")
-    print("-----------------------------")
+    logger.info("-----------------------------")
+    logger.info(f"Flux Overkill: {game_vars.flux_overkill()}")
+    logger.info("Seymour Flux battle complete.")
+    logger.info("-----------------------------")
     # time.sleep(60) #Testing only
 
+def s_keeper_bahamut_crit() -> int:
```

Comment:
+1 refactor

Reply:
Yeah just saw that repeated a bunch of times and couldn't help myself :)